src/components/CreatePostInput/index.js

```diff
@@ -214,14 +199,9 @@ const CreatePostInput = ({
         handleStateHashtag={handleStateHashtag}
         hashtagPosition={hashtagPosition}
         positionTopicSearch={positionTopicSearch}
-        // setHashtags={setHashtags}
         setPositionKeyboard={setPositionKeyboard}
         setMessage={setMessage}
-        setTopicChats={setTopicChats}
         setTopicSearch={setTopicSearch}
-        setTopics={setTopics}
-        topicChats={topicChats}
-        topics={topics}
         topicSearch={topicSearch}
       />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several props have been removed from this component. If these props are no longer used by the child component, this change is fine. However, if the child component still relies on these props, they should be added back.
```diff
+         setHashtags={setHashtags}
+         setTopicChats={setTopicChats}
+         setTopics={setTopics}
+         topicChats={topicChats}
+         topics={topics}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->